Cargo.toml

```diff
@@ -12,16 +12,13 @@ keywords = ["proc-macro", "error", "errors"]
 categories = ["development-tools::procedural-macro-helpers"]
 license = "MIT OR Apache-2.0"
 edition = "2021"
+build = "build.rs"
```

Comment:
Is this necessary? I was under the impression this was automatically done.

Reply:
It is automatically done. It's once again a style question and I'm happy to remove if you'd like.

Reply:
Please do.